cernopendata/modules/fixtures/data/terms/terms.json

```diff
@@ -12746,5 +12746,328 @@
+        "url": "https://aleph.web.cern.ch"
+      }
+    ],
+    "see_also": [
+      {
+        "term": "LEP"
+      },
+      {
+        "term": "DELPHI"
+      },
+      {
+        "term": "L3"
+      },
+      {
+        "term": "OPAL"
+      }
+    ],
+    "term": [
+      "ALEPH"
+    ],
+    "experiment": ["ALEPH", "DELPHI", "L3", "OPAL"],
+    "type": {
+      "primary": "Glossary"
+    }
+  },
+  {
+    "anchor": "DELPHI",
+    "category": "generic",
+    "definition": "DEtector with Lepton, Photon and Hadron Identification. DELPHI was one out of four experiments of the LEP collider",
+    "links": [
+      {
+        "text": "DELPHI home page",
+        "url": "https://delphi-www.cern.ch"
+      }
+    ],
+    "see_also": [
+      {
+        "term": "LEP"
+      },
+      {
+        "term": "ALEPH"
+      },
+      {
+        "term": "L3"
+      },
+      {
+        "term": "OPAL"
+      }
+    ],
+    "term": [
+      "DELPHI"
+    ],
+    "experiment": ["ALEPH", "DELPHI", "L3", "OPAL"],
+    "type": {
+      "primary": "Glossary"
+    }
+  },
+  {
+    "anchor": "OPAL",
+    "category": "generic",
+    "definition": "Omni-Purpose Apparatus at LEP. OPAL was one out of four experiments of the LEP collider",
+    "links": [
+      {
+        "text": "OPAL home page",
+        "url": "https://opal.web.cern.ch"
+      }
+    ],
+    "see_also": [
+      {
+        "term": "LEP"
+      },
+      {
+        "term": "ALEPH"
+      },
+      {
+        "term": "DELPHI"
+      },
+      {
+        "term": "L3"
+      }
+    ],
+    "term": [
+      "OPAL"
+    ],
+    "experiment": ["ALEPH", "DELPHI", "L3", "OPAL"],
+    "type": {
+      "primary": "Glossary"
+    }
+  },
+  {
+    "anchor": "L3",
+    "category": "generic",
+    "definition": "L3 was one out of four experiments of the LEP collider",
+    "links": [
+      {
+        "text": "L3 home page",
+        "url": "https://l3.web.cern.ch"
+      }
+    ],
+    "see_also": [
+      {
+        "term": "LEP"
+      },
+      {
+        "term": "ALEPH"
+      },
+      {
+        "term": "DELPHI"
+      },
+      {
+        "term": "OPAL"
+      }
+    ],
+    "term": [
+      "L3"
+    ],
+    "experiment": ["ALEPH", "DELPHI", "L3", "OPAL"],
+    "type": {
+      "primary": "Glossary"
+    }
+  },
+  {
+    "anchor": "ZEBRA",
+    "category": "generic",
+    "definition": "ZEBRA is a data format which implements dynamic data structures for Fortran77.",
+    "links": [
+      {
+        "text": "ZEBRA manual",
+        "url": "https://cds.cern.ch/record/2296399/files/zebra.pdf"
+      }
+    ],
+    "see_also": [
+      {
+        "term": "CERNLIB"
+      },
+      {
+        "term": "DELPHI"
+      },
+      {
+        "term": "OPAL"
+      },
+      {
+        "term": "JADE"
+      }
+    ],
+    "term": [
+      "ZEBRA"
+    ],
+      "experiment": ["DELPHI", "L3", "OPAL", "JADE"],
+    "type": {
+      "primary": "Glossary"
+    }
+  },
+  {
+    "anchor": "PAW",
```

Comment:
For consistency, since PAW is described, what about adding also GEANT?

Reply:
Hmm, these are two quite different pairs of shoes :) DELPHI didn't use GEANT (to the best I know at least). We can add it once OPAL joins the club I'd say ?

Reply:
OK, I see why you want it. Because I quoted it myself above. Right. Good point.